## 01_automake1.10.1.diff by Ryan Kavanagh <ryanakca@kubuntu.org>
## Description: Allow us to use automake*1.10*
--- a/admin/cvs.sh
+++ b/admin/cvs.sh
@@ -64,7 +64,8 @@ case $AUTOMAKE_STRING in
     echo "*** KDE requires automake 1.6.1 or newer"
     exit 1
     ;;
-  automake*1.6.* | automake*1.7* | automake*1.8.* | automake*1.9* ) : ;;
+  automake*1.6.* | automake*1.7* | automake*1.8.* | automake*1.9* | \
+  automake*1.10* ) : ;;
   "" )
     echo "*** AUTOMAKE NOT FOUND!."
     echo "*** KDE requires automake 1.6.1 or newer"
